### PR TITLE
move fluid.initializer.set_global_initializer to nn.initializer

### PR DESCRIPTION
**Pull request by windstamp**
### PR types

New features
### PR changes

APIs
### Describe

move fluid.initializer.set_global_initializer to nn.initializer

### CODE REVIEW

**Comment by windstamp** [comment]
image

File: python/paddle/nn/initializer/initializer.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def set_global_initializer(weight_init, bias_init=None):
+    """
+    This API is used to set up global model parameter initializer in framework.
+
+    After this API is invoked, the global initializer will takes effect in subsequent code.
+
+    The model parameters include ``weight`` and ``bias`` . In the framework, they correspond 
+    to ``fluid.Parameter`` , which is inherited from ``fluid.Variable`` , and is a persistable Variable.
```
Comment (TCChenlong) [inline python/paddle/nn/initializer/initializer.py]: fluid.Parameter -> paddle.ParamAttr
fluid.Variable -> paddle.Tensor

File: python/paddle/nn/initializer/initializer.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        .. code-block:: python
+            import paddle.fluid as fluid
+            import paddle.nn as nn
+
+            nn.initializer.set_global_initializer(nn.initializer.Uniform(), nn.initializer.Constant())
+            x = paddle.static.data(name="x", shape=[1, 3, 32, 32])
+
+            # The weight of conv1 is initialized by Uniform
+            # The bias of conv1 is initialized by Constant
+            conv1 = fluid.layers.conv2d(x, 5, 3)
```
Comment (TCChenlong) [inline python/paddle/nn/initializer/initializer.py]: paddle.nn.functional.conv2d

File: python/paddle/nn/initializer/initializer.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+            # If set param_attr/bias_attr too, global initializer will not take effect
+            # The weight of conv2 is initialized by Xavier
+            # The bias of conv2 is initialized by Normal
+            conv2 = fluid.layers.conv2d(conv1, 5, 3,
```
Comment (TCChenlong) [inline python/paddle/nn/initializer/initializer.py]: 同上

File: python/paddle/nn/initializer/initializer.py (hunk 1 above)
Comment (windstamp) [reply python/paddle/nn/initializer/initializer.py]: thanks, done.

File: python/paddle/nn/initializer/initializer.py (hunk 2 above)
Comment (windstamp) [reply python/paddle/nn/initializer/initializer.py]: thanks, done.

File: python/paddle/nn/initializer/initializer.py (hunk 3 above)
Comment (windstamp) [reply python/paddle/nn/initializer/initializer.py]: thanks, done.

**Pull request closed without merging**